src/pages/Contributors/AffiliatedOrganizations.js

```diff
@@ -66,65 +66,84 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export const AffiliatedOrganizations = ({ organizations , inputValue, data, checkboxValue }) => {
-
+export const AffiliatedOrganizations = ({
+  organizations,
+  inputValue,
+  organizationData,
+  checkboxValue,
+}) => {
   const classes = useStyles();
-  const parentfilterData = data;
-  const [isChildThumbnail] = useState(true);
+  const parentfilterData = organizationData;
```

Comment:
Why do you create a new variable called `parentfilterData`? Nothing happens to it, so it seems unnecessary, unless I'm missing something ....

Reply:
Thanks Kevin will make the changes

Reply:
I am using parentfilterData to mapping child to parent. If i give organizationData.find , it is giving error TypeError: organizationData.find is not a function.

Reply:
> I am using parentfilterData to mapping child to parent. If i give organizationData.find , it is giving error TypeError: organizationData.find is not a function.

That is because you're using the same variable name in 2 places.

```
export const AffiliatedOrganizations = ({
  organizations,
  inputValue,
  organizationData,
  checkboxValue,
}) => {
```
```
organizationArray.forEach((organizationData) => {
```

Like Kevin mentioned, you should use different variable names so that you don't get confused as to which one you're using at any place in the code.

Reply:
As suggested earlier, use org instead of organizationData inside the forEach loop